lms/djangoapps/certificates/models.py

```diff
@@ -466,6 +467,10 @@ def certificate_status_for_student(student, course_id):
                    should not be issued a certificate. This will
                    be set if allow_certificate is set to False in
                    the userprofile table
+    unverified   - The student was in verified enrollment track and
```

Comment:
Reading the comment, it's not really clear to me here why honor mode has anything to do with unverified status.  Plus the past tense of "was in verified enrollment track" leads me to believe they are no longer in said track -- can we clarify the path a student must follow in order to end up with an 'unverified' status?